public/app/components/users.dash/user-manager.tpl.html

```diff
@@ -7,15 +7,15 @@
       <p class="md-body-2" translate="{{'NAME'}}" flex="25"></p>
       <p class="md-body-2" translate="{{'IDENTITY'}}" flex="25"></p>
       <p class="md-body-2" translate="{{'ROLES'}}" flex="25"></p>
-      <p class="md-body-2" flex="25">Pending verification</p>
+      <p class="md-body-2" translate="{{'ADDRESS'}}" flex="25"></p>
```

Comment:
You should namespace this, like it's supposed to be "Address Verified" right? Translation tokens should really be the full english sentence, I'd expect this one to be "Address".